source/whats-new.txt

```diff
@@ -350,7 +350,7 @@ New features of the 4.10 {+driver-short+} release include:
          myColl.findOne().then(res => callback(null, res), err => callback(err))
 
   - If you are unable to migrate to Promises in a large codebase, you can
-    use the {+node-legacy+} package.
+    use the `{+driver-long+} with optional callback support <https://www.npmjs.com/package/mongodb-legacy>`__ package.
```

Comment:
Suggestion:
Not certain whether this is in the style guide, but our team decided that we only use the "driver-long" form in the first mention of the driver on the page, and only when in introductory material. While this is referring to a legacy version of it, the name is exactly the same and it would be repetitive as well since line 331 already introduces the bulleted list by using "driver-short".